src/dg/strong_dg.cpp

```diff
@@ -539,7 +551,7 @@ void DGStrong<dim,nstate,real,MeshType>::assemble_volume_term_explicit(
     std::vector< realArrayTensor1 > conv_phys_flux_at_q(n_quad_pts);
     std::vector< realArrayTensor1 > diss_phys_flux_at_q(n_quad_pts);
     std::vector< realArray > source_at_q(n_quad_pts);
-
+    std::vector< realArray > physical_source_at_q(n_quad_pts);
```

Comment:
Declare but allocate only if `this->pde_physics_double->has_nonzero_physical_source`

Reply:
fixed.
It is now only declared and allocate (resize) when has_nonzero_physical_source is true.

Reply:
The same work was done for the source_at_q.